### PR TITLE
Make python version of JobPosting

### PR DESCRIPTION
**Pull request by fbjorn**
Added python version for JobPosting. Translated some descriptions to English.
You can compare before/after on the following screenshots:
image
image

### CODE REVIEW

**Comment by fbjorn** [comment]
Hi @lassipatanen, we're changing the repo structure to make all definitions have python version as an original file, because it's easier to maintain. Would be nice if you could also check the changes, the resulting `.json` file should be the same except of some rewordings or extra fields for titles and descriptions

**Comment by lassipatanen** [comment]
@lsipii ping

**Comment by lsipii** [comment]
@fbjorn Looks good to me!

File: src/test/lassipatanen/Job/JobPosting.py [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    )
+    description: str = Field(
+        ...,
+        title="Description",
+        description="Short description about the job",
+    )
+    work_time_type: Optional[str] = Field(
+        None,
+        title="Work time type",
+        description="01 - Full time, 02 - Part time",
```
Comment (joakimnordling) [inline src/test/lassipatanen/Job/JobPosting.py:71]: This one is also missing the example and the pre-formatted description has changed.
image

```suggestion
        description="<pre>01 = Full time\n02 = Part time</pre>",
        example="01",
```

**Review by joakimnordling — APPROVED** [review]
LGTM

**Pull request merged**